Sources/GATTBodyCompositionMeasurement.swift

```diff
@@ -652,6 +655,8 @@ extension GATTBodyResistance: CustomStringConvertible {
 
 public struct GATTBodyMass {
```

Comment:
@JurjDev are these structs being used in the places?, if not we should move them inside GATTBodyCompositionMeasurement and remove GATT prefix